include/fplus/sets.hpp

```diff
@@ -31,12 +31,9 @@ bool set_includes(const SetType& set1, const SetType& set2)
 template <typename UnorderedSetType>
 bool unordered_set_includes(const UnorderedSetType& set1, const UnorderedSetType& set2)
 {
-    for(const auto& x : set2)
-    {
-        if (set1.find(x) == set1.end())
-            return false;
-    }
-    return true;
+    auto first_not_included = std::find_if(set2.begin(), set2.end(),
+                                           [&](const typename UnorderedSetType::value_type& x) -> bool { return set1.find(x) == set1.end();});
```

Comment:
style: this line goes over the Github diff window.

Also, there's no need to the lambda argument type or return type (supposing we do C++14):

    [&](auto&& x) { return set1.find(x) == set1.end();};